tests/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Looks like this was just copied from the PlaywrightFramework repo. The better way is to use yarn or npm and initialize a new package.json in the main directory. Happy to jump on a call to walk you through that or just do it and push it up